scripts/modules/elastic_stack.py

```diff
@@ -480,7 +482,11 @@ def _content(self):
             build_spec_parts = self.build.split("@", 1)
             repo = build_spec_parts[0]
             branch = build_spec_parts[1] if len(build_spec_parts) > 1 else "master"
-            binary = "apm-server-oss" if self.oss else "apm-server"
+            binary = "apm-server"
+            if self.oss:
+                binary = "apm-server-oss"
+            if self.ubi8:
+                binary = "apm-server-ubi8"
```

Comment:
```suggestion
            binary = "apm-server-oss" if self.oss else "apm-server"
```

The apm-server binary name does not change for the ubi8 images; only for OSS.